src/main/java/org/opensearch/security/dlic/rest/api/AccountApiAction.java

```diff
@@ -128,20 +145,28 @@ protected void handleGet(RestChannel channel, RestRequest request, Client client
         try {
             builder.startObject();
             final User user = threadContext.getTransient(ConfigConstants.OPENDISTRO_SECURITY_USER);
-            if (user != null) {
+            final SecurityDynamicConfiguration<?> internalUser = load(CType.INTERNALUSERS, false);
+
+            if (user != null){
                 final TransportAddress remoteAddress = threadContext.getTransient(ConfigConstants.OPENDISTRO_SECURITY_REMOTE_ADDRESS);
                 final Set<String> securityRoles = privilegesEvaluator.mapRoles(user, remoteAddress);
-                final SecurityDynamicConfiguration<?> configuration = load(getConfigName(), false);
-
+            
                 builder.field("user_name", user.getName())
-                        .field("is_reserved", isReserved(configuration, user.getName()))
-                        .field("is_hidden", configuration.isHidden(user.getName()))
-                        .field("is_internal_user", configuration.exists(user.getName()))
-                        .field("user_requested_tenant", user.getRequestedTenant())
-                        .field("backend_roles", user.getRoles())
-                        .field("custom_attribute_names", user.getCustomAttributesMap().keySet())
-                        .field("tenants", privilegesEvaluator.mapTenants(user, securityRoles))
-                        .field("roles", securityRoles);
+                    .field("is_reserved", isReserved(internalUser, user.getName()))
+                    .field("is_hidden", internalUser.isHidden(user.getName()))
```

Comment:
There are no changesi n builder. We should consider reverting back unnecesaary changes. It increases diff and gets unnecessary attention